docs/index.md

```diff
@@ -15,4 +15,4 @@ Couleur Tweak Tips (CTT) is a Discord server and wiki, dedicated to:
 * Configuring operating systems: Windows/Linux' appearance & behavior, keyboard shortcuts
 * Quality of life software: ShareX, AltSnap, PowerToys, SpotX, MAS, espanso, SearchEverything
 
-Browse through the left pane to get started
\ No newline at end of file
+To get started, browse the wiki using the sidebar on the left, or use the search bar at the top of the page.
```

Comment:
it'd be at the top in most cases, not the left (unless it's mobile)
![librewolf_BP9ro0rIN0](https://github.com/couleur-tweak-tips/CTT/assets/65787561/f8b99743-155b-4068-af5d-b068bb9338b2)

Reply:
That's a change made by couleur iirc and I'm not messing with changes by the big man

Reply:
he's wrong dw :3

Reply:
i am even tho i'm big 👍